### PR TITLE
Disable CG Check for Partner Release Pipelines

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Add flag for skipping Component Governance steps

### CODE REVIEW

File: eng/pipelines/templates/jobs/azuresdkpartnerdrops-to-nugetfeed.yml [hunk 1]
```diff
@@ -44,6 +44,10 @@ parameters:
     type: boolean
     default: true
 
+variables:
+  - name: skipComponentGovernanceDetection
```
Comment (benbp) [inline eng/pipelines/templates/jobs/azuresdkpartnerdrops-to-nugetfeed.yml:48]: Would it be better to import [globals.yml](https://github.com/Azure/azure-sdk-tools/blob/main/eng/pipelines/templates/variables/globals.yml) here or will that add too many other conflicting variable overrides?

Comment (chidozieononiwu) [reply eng/pipelines/templates/jobs/azuresdkpartnerdrops-to-nugetfeed.yml:48]: It should be fine to import globals.yml

**Review by weshaggard — APPROVED** [review]

**Pull request merged**